src/openrct2/paint/Paint.h

```diff
@@ -51,6 +51,15 @@ enum PAINT_QUADRANT_FLAGS {
     PAINT_QUADRANT_FLAG_NEXT = (1 << 1),
 };
 
+typedef struct paint_struct_bound_box {
+    uint16 x;
+    uint16 y;
+    uint16 z;
+    uint16 z_end;
```

Comment:
I don't think there is any reason why you need to keep this funky order. X, y, z would be fine

Reply:
@duncanspumpkin good catch. I wanted to include this too, but have sadly forgotten.